mappings/net/minecraft/client/render/entity/feature/HorseMarkingRenderer.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_5167 net/minecraft/client/render/entity/feature/HorseMarkingRenderer
```

Comment:
Shouldn't it be `HorseMarkingFeatureRenderer`?

Reply:
Done.